tests/unittests/test_commands.py

```diff
@@ -105,5 +105,18 @@ def test_text_saves_params(self):
         self.assertEqual(output_text.fontsize, 64)
 
 
+class TestFontUtils(ShoebotTestCase):
+    @test_as_bot()
+    def test_fontnames_gives_output(self):
+        """
+        Verify that fontnames() gives a list as output.
+        """
+        output = fontnames()
+        self.assertIsInstance(output, list)
+        self.assertTrue(output)
+        # contains a string
+        self.assertIsInstance(output[0], str)
```

Comment:
This is looking good, only thing I can think is we could add a check that the format is good: and have:

```assertRegexpMatches(output[0], r"(.*)\s(.*)")```

It's not the most exhaustive test, just that there is a space (we could improve it later though) - but then it's good to merge - thanks for taking the time to get this to the end :)